### PR TITLE
Allow retrieving credentials for all users

### PR DESCRIPTION
**Pull request by icewind1991**
cc @PVince81 @jvillafanez

### CODE REVIEW

File: tests/lib/Security/CredentialsManagerTest.php [hunk 1]
```diff
@@ -57,6 +57,18 @@ private function getQeuryResult($row) {
 		return $result;
 	}
 
+	private function getQeuryResultMultiple($rows) {
```
Comment (nickvergessen) [inline tests/lib/Security/CredentialsManagerTest.php]: typo in `qEury`

Comment (nickvergessen) [inline tests/lib/Security/CredentialsManagerTest.php]: also you dont use this?

Comment (georgehrke) [inline tests/lib/Security/CredentialsManagerTest.php]: ping

**Comment by georgehrke** [comment]
Is there an related issue?

**Comment by icewind1991** [comment]
This came from an EE issue but it seems to be no longer needed, closing for now

**Comment by DeepDiver1975** [comment]
> e but it seems to be no longer needed, closing for now

why?

**Comment by jvillafanez** [comment]
Information about the host was needed to resolve the issue. Although we could get the credentials for several user, is wasn't possible to know which host are those credentials for, and that information isn't saved as part of the credentials.

**Pull request closed without merging**